packages/shared/src/storage.ts

```diff
@@ -0,0 +1,107 @@
+/* -----------------------------------
+ *
+ * Helpers
+ *
+ * -------------------------------- */
+
+type StorageClassName = 'localStorage' | 'sessionStorage';
+
+
+function getRootObject(): Window | undefined {
+   if (typeof self === 'object') {
+      return self;
+   } else if (typeof window === 'object') {
+      return window;
+   }
+}
```

Comment:
May [`globalThis`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis) be used here? It is designed for accessing the global this object across the client, web workers, etc.

Reply:
Migrated to `globalThis` in the latest commit